zaza/openstack/charm_tests/ceph/rbd_mirror/tests.py

```diff
@@ -166,6 +184,18 @@ def setUpClass(cls):
         # get ready for multi-model Zaza
         cls.site_a_model = cls.site_b_model = zaza.model.get_juju_model()
         cls.site_b_app_suffix = '-b'
+        # Test if we have the needed Openstack applications.
+        try:
+            zaza.model.get_application(cls.cinder_ceph_app_name)
+            cls.with_cinder = True
+        except Exception:
+            cls.with_cinder = False
+
+    def check_cinder_present(self, caller):
+        """Skip a test if Cinder isn't present."""
```

Comment:
I think that this would be better named something like `skip_test_if_cinder_is_not_present()` as that is what it actually does.  At the moment, it looks like a predicate that has an unexpected side-effect.

Reply:
Done.